arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -361,41 +361,48 @@
       "required": [ "Name" ],
       "properties": {
         "Name": {
-          "description": "Name of the offer term. For example: 'Monetary Credit', 'Monetary Commitment",
-          "type": "string"
+          "description": "Name of the offer term",
+          "type": "string",
+          "enum": [
+            "Recurring Charge",
+            "Monetary Commitment",
+            "Monetary Credit"
+          ]
```

Comment:
Please use the extension ["x-ms-enum"](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md#x-ms-enum).